AppcuesCapacitor.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'Appcues', '~> 1.1'
+  s.dependency 'Appcues', '~> 1.1.1'
```

Comment:
Worth noting that this means 1.1.x only whereas what we had before would include any 1.x, so a theoretical iOS SDK 1.2 means we'd need to do a new Capacitor release.

(reference: https://guides.cocoapods.org/using/the-podfile.html#specifying-pod-versions)

Reply:
Right, yeah meant to comment on this. I _think_ its necessary, since the new `config.enableUniversalLinks` is only available on 1.1.1 and up, and if you simply upgraded the capacitor plugin but did not force that minimum dependency version, it would have a build issue. Do you see any better approach?

Reply:
Yeah, it has to be like this. No better approach that I'm aware of. We just need to be aware what's included for future version releases.

I think in general we'll want to sync xplat versions for major and minor versions if only to make docs and communicating with customers easier (ie "use 1.1 for surveys"). Patch versions can get out sync and that doesn't matter.